internal/expr/expr_test.go

```diff
@@ -41,8 +41,20 @@ type HardMaths struct {
 	Coef int `db:"coef"`
 }
 
+type QuotedColumnNames struct {
+	Ex int `db:"\"!!!\""`
+	Qu int `db:"'???'"`
+}
+
+type NumberLiteralColumn struct {
+	Zero int `db:"0"`
+	One  int `db:"1"`
+}
+
 type M map[string]any
 
+type lowerCaseMap map[string]any
```

Comment:
I don't quite follow what's lowercase here, is this testing unexported types? In this case I think `unexportedMap` would be more descriptive

nit: I think the 'C' in the name should be lowercase.

Reply:
Good point, will change it to that.